onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -130,13 +134,21 @@ def shape(self, shape: Tuple[int | None, ...]):
     @property
     def dtype(self):
         # TODO: Return numpy dtype
-        return _type_utils.JitScalarType.from_value(  # type: ignore[attr-defined]
+        torch_dtype = _type_utils.JitScalarType.from_value(  # type: ignore[attr-defined]
```

Comment:
I see, you don't have a `torch.dtype` for sequence to be passed in from `dtype.setter` to begin with.

Reply:
Frankly feels it's a bit overloaded and doesn't seem to match the design for this class, to be also covering sequence type.

Reply:
Should we add another attribute?

Reply:
One of alternative way that I can think of is that we give up this dtype, and try to relax `match_schema` in a way that it finds the best match instead of "the 100% match", as I found some inputs are just None anyway.

Reply:
Reverted